src/modules/luksbootkeyfile/LuksBootKeyFileJob.cpp

```diff
@@ -297,8 +323,8 @@ LuksBootKeyFileJob::exec()
 
         if ( !setupLuks( d ) )
             return Calamares::JobResult::error(
-                tr( "Encrypted rootfs setup error" ),
-                tr( "Could not configure LUKS key file on partition %1." ).arg( d.device ) );
+                tr( "Encryption setup error" ),
+                tr( "Could not configure LUKS on partition %1." ).arg( d.device ) );
```

Comment:
I think the prior error was more descriptive.

Reply:
I changed the error message to distinct between two identical error messages. So, it would be easier to see which code threw the error.

Reply:
The prior message noted that the failure was related to the key file and the rootfs.  Now it is either overly generic or misleading depending on how you read it.